ipaserver/install/service.py

```diff
@@ -157,7 +158,8 @@ def ldap_disconnect(self):
         LDAPConnectionManager.close_connection()
         self.admin_conn = None
 
-    def _ldap_mod(self, ldif, sub_dict=None, raise_on_err=True):
+    def _ldap_mod(self, ldif, sub_dict=None, raise_on_err=True,
```

Comment:
I don't like to ahve so much options, can we split this into separate method _simple_ldap_mod(), _nonldapi_, or something